homeassistant/components/home_connect/api.py

```diff
@@ -348,31 +400,47 @@ class Oven(
     DeviceWithPrograms,
     DeviceWithRemoteControl,
     DeviceWithRemoteStart,
+    DeviceWithLocalControl,
+    DeviceWithProgramFinished,
 ):
     """Oven class."""
 
-    PROGRAMS = [
-        {"name": "Cooking.Oven.Program.HeatingMode.PreHeating"},
```

Comment:
Removing this looks like a breaking change. Is it that?

Reply:
Formally - yes, we remove programs switches from ovens. But this switches didn't work. So, I don't know is it breaking change or not.
Can you give me any advice?

Reply:
I think it's better to remove this in a separate PR.